requirements.txt

```diff
@@ -1,10 +1,10 @@
-django>=1.11,<2.2
+django>=1.11,<3.0
 django-model-utils
 jsonfield
 six
 netjsonconfig>=0.6.2,<0.7.0
-django-sortedm2m>=1.5.0,<1.6
+django-sortedm2m>=1.6.0,<2.1.0
```

Comment:
Check `sortedm2m` requirements [here](https://requires.io/github/openwisp/django-netjsonconfig/requirements/?branch=master).
There no version `1.6.0` nor `2.1.0`, so we support only version `2.0.0` for this.
@nemesisdesign since only one version will work for this update it's suppose to be `django-sortedm2m==2.1.0` right?

Reply:
Done :smile:

Reply:
Please mark done after pushing the changes! :smile: